core/src/net/sf/openrocket/rocketcomponent/FinSet.java

```diff
@@ -1002,26 +1029,85 @@ public Coordinate[] getTabPoints() {
-			return new Coordinate( 0, 0);
-		}else{
-			final double yFinFront = symmetricParent.getRadius( xFinFront );
-			return new Coordinate(xFinFront, yFinFront);
-		}
-	}
-
-
-	/* 
-	 * yes, this may over-count points between the fin and fin tabs, 
+	/*
+	 * yes, this may over-count points between the fin and fin tabs,
 	 * but the minor performance hit is not worth the code complexity of dealing with.
 	 */
 	public Coordinate[] getFinPointsWithTab() {
 		Coordinate[] temp = combineCurves(getFinPoints(), getRootPoints());
 		return combineCurves(temp, getTabPoints());
 	}
+
+	/**
+	 * use this for calculating physical properties, and routine drawing
+	 *
+	 * @return points representing the fin-root points, relative to ( x: fin-front, y: centerline ) i.e. relto: fin Component reference point
+	 */
+	public Coordinate[] getMountPoints() {
+		if( null == parent){
+			return null;
+		}
+
+		return getMountPoints(0., parent.getLength(), 0,0);
+	}
+
+	/**
+	 * used to get calculate body profile points:
+	 *
+	 * @param xStart - xStart, in Mount-frame
+	 * @param xEnd - xEnd, in Mount-frame
+	 * @param xOffset - x-Offset to apply to returned points
+	 * @param yOffset - y-Offset to apply to returned points
+	 *
+	 * @return points representing the mount's points
+	 */
+	private Coordinate[] getMountPoints(final double xStart, final double xEnd, final double xOffset, final double yOffset) {
+		if (parent == null) {
+			return new Coordinate[]{Coordinate.ZERO};
+		}
+
+		// for a simple body, one increment is perfectly accurate.
+		int divisionCount = 1;
+		final SymmetricComponent body = (SymmetricComponent) getParent();
+		final double intervalLength = xEnd - xStart;
+
+		// for anything more complicated, increase the count:
+		if ((body instanceof Transition) && (((Transition)body).getType() != Shape.CONICAL)) {
+			// the maximum precision to enforce when calculating the areas of fins (especially on curved parent bodies)
+			final double xWidth = 0.0025; // width (in meters) of each individual iteration
+			divisionCount = (int) Math.ceil(intervalLength / xWidth);
+
+			// When creating body curves, don't create more than this many divisions. -- only relevant on very large components
+			final int maximumBodyDivisionCount = 100;
+			divisionCount = Math.min(maximumBodyDivisionCount, divisionCount);
+		}
+
+		// Recalculate the x step increment, now with the (rounded) division count.
+		double xIncrement = intervalLength / divisionCount;
+
+		// Create the points: step through the radius of the parent
+		double xCur = xStart;
+		Coordinate[] points = new Coordinate[divisionCount+1];
+		for (int index = 0; index < points.length; index++) {
+			double yCur = body.getRadius(xCur);
+			points[index] = new Coordinate(xCur, yCur);
+
+			xCur += xIncrement;
+		}
+
+		// correct last point, if beyond a rounding error from body's end.
+		final int lastIndex = points.length - 1;
+		if (Math.abs(points[lastIndex].x - body.getLength()) < 0.000001) {
```

Comment:
Is there a reason why the rounding error here is 1x10<sup>-6</sup> instead of `MathUtil.EPSILON` (1x10<sup>-8</sup>)? I believe that MathUtil.EPSILON is generally treated as the rounding error throughout (including unit tests and such) - so it would probably be better to be consistent (or note the reason for deviation).

Reply:
Just a copy-paste from previous code that was not written by me. I'll adjust it.